services/core-api/app/api/incidents/models/mine_incident.py

```diff
@@ -74,33 +75,33 @@ class MineIncident(AuditMixin, Base):
 
     determination_type = db.relationship(
         'MineIncidentDeterminationType',
-        backref='mine_incidents',
+        backref='mine_incident',
```

Comment:
Pretty sure these were typos

Reply:
These are not typos. this defines the reverse relationship from the target object to this object. 

for example, we could go determination_type.mine_incidents and get all the incidents that use that code, so it is plural. Backref is weird and I think long term we are going to phase it out because it's tough to track what relationships exist. for now just revert this change.

Reply:
I see. Are we even making use of these backrefs in our project?